components/shared/upload/DropUpload.vue

```diff
@@ -0,0 +1,197 @@
+                  {{ file.name }} {{ $t('uploaded') }}
+                </span>
+                <div class="upload-file-label">
+                  {{ $t('chooseFile') }}
+                </div>
+              </div>
+            </div>
+          </section>
+        </o-upload>
+      </b-field>
+      <transition v-if="fileSizeFailed" name="fade">
+        <div class="help is-danger">
+          {{ $t('tooltip.failedMaxSize') }}
+        </div>
+      </transition>
+      <transition v-if="checkFailed" name="fade">
+        <div class="help is-danger">
+          {{ $t('tooltip.needToUploadNFTFile') }}
+        </div>
+      </transition>
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { MAX_UPLOADED_FILE_SIZE } from '@/utils/constants'
+import { OUpload } from '@oruga-ui/oruga'
+import { NeoIcon } from '@kodadot1/brick'
+
+const MediaResolver = defineAsyncComponent(
+  () => import('@/components/media/MediaResolver.vue')
+)
+
+const props = withDefaults(
+  defineProps<{
+    label?: string
+    icon?: string
+    required?: boolean
+    preview?: boolean
+    expanded?: boolean
+    accept?: string
+    format?: string
+  }>(),
+  {
+    label:
+      'Drop your NFT here or click to upload or simply paste image from clipboard',
+    icon: 'cloud-arrow-up',
+    required: false,
+    preview: false,
+    expanded: false,
+    accept: '',
+    format: '',
+  }
+)
+
+const file: any = ref(null)
+const url = ref('')
+
+const fileSizeLimit = MAX_UPLOADED_FILE_SIZE
+let hasError = false
+let checkFailed = false
+let fileSizeFailed = false
+
+const emit = defineEmits(['input'])
+
+const mimeType = computed(() => {
+  if (file?.value?.type) {
+    return file?.value.type
+  }
+
+  const fileExtension = file?.value?.name.split('.').pop() || ''
+  const extensionToMimeType = {
+    glb: 'model/gltf-binary',
+  }
+  if (fileExtension in extensionToMimeType) {
+    return extensionToMimeType[fileExtension]
+  }
+  return ''
+})
+
+const isModelMedia = computed(() => {
+  return mimeType.value.startsWith('model')
+})
+
+const createInput = (file): void | boolean => {
+  const fileSize = file.size / Math.pow(1024, 2)
+  if (fileSize > fileSizeLimit) {
+    fileSizeFailed = true
+    file = null
+    return false
+  }
+  fileSizeFailed = false
+  checkFailed = false
+  const reader = new FileReader()
+  reader.onload = () => {
+    // onload
+  }
+  emit('input', file)
+  if (props.preview) {
+    url.value = URL.createObjectURL(file)
+    hasError = false
+  }
+
+  reader.readAsText(file)
+}
+</script>
+<style scoped lang="scss">
+@import '@/styles/abstracts/variables';
+.cloud-icon {
+  font-size: 55px;
+  @include ktheme() {
+    color: theme('k-grey');
+  }
+}
+
+.check-icon {
+  font-size: 42px;
+  @include ktheme() {
+    color: theme('k-green');
+  }
+}
+
+.description {
+  @include ktheme() {
+    color: theme('k-grey');
+  }
+}
+
+.main-label {
+  max-width: 390px;
+}
+
+.labels {
+  text-align: left;
+}
```

Comment:
```suggestion

```
Is it necessary? `left` is the default value of text-align

Reply:
yes but here the whole container has align items center and justify content center, so it is required, without this, it would be center aligned

Reply:
Ok, then let's use `has-text-left` instead